packages/app-mobile/utils/fs-driver/fs-driver-rn.ts

```diff
@@ -23,19 +55,26 @@ export default class FsDriverRN extends FsDriverBase {
 		throw new Error('Not implemented');
 	}
 
-	// Encoding can be either "utf8" or "base64"
-	public appendFile(path: string, content: any, encoding = 'base64') {
+	// Requires that the file already exists.
+	// TODO: Update for compatibility with fs-driver-node's appendFile (which does not
+	//       require that the file exists).
+	public appendFile(path: string, content: any, rawEncoding = 'base64') {
```

Comment:
While writing tests, I found the issue mentioned in the above comment.

I think addressing this is outside of the current scope of this pull request. Would it make sense to expand the pull request's scope and address it here?